app/dashboard/src/components/AriaComponents/Form/Form.tsx

```diff
@@ -80,6 +78,17 @@ export const Form = React.forwardRef(function Form<
     },
   )
 
+  // Expose default values to controlled inputs like `Selector` and `MultiSelector`
+  const initialDefaultValues = React.useRef(defaultValues)
+  React.useEffect(() => {
```

Comment:
This shall be moved to the `useForm` hook instead.
Also, I guess we might want to reset only the unchanged fields?

Reply:
tried changing `form.reset(defaults)` to `form.reset(defaults, { keepDirtyValues: true })` but that's causing the timeout in minutes value (60) to not appear for some reason. maybe we're not managing the dirty state properly